src/components/ButtonPanel.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Button from './Button';
+
+
+class ButtonPanel extends React.Component {
+constructor () {
+    super();
+    this.state = {
+      panelElm : ['AC', '+/-', '%', '+',
+                  '7', '8', '9', 'X',
+                  '4', '5', '6', '-',
```

Comment:
Hey, this is not gonna change, we don't necessarily need it in state, it can just be an array in render.